app/views/trips/_trip.html.haml

```diff
@@ -1,5 +1,5 @@
 .trip
-  = link_to trip, class: 'trip-item' do
+  = link_to trip, class: [('trip-item-before' if ((trip.departure_date == Date.today && trip.before_actual_time) || trip.departure_date < @search.date_value)), ('trip-item-after' if trip.departure_date > @search.date_value), ('trip-item')] do
```

Comment:
Might be proper to create `Trip.is_before_today?` and `Trip.is_after_today?` Model method check to reuse it somewhere else.